frontend/src/views/CompanyLogin.vue

```diff
@@ -0,0 +1,60 @@
+<template>
+  <div class="companyLogin">
+    <Header />
```

Comment:
Skal denne headeren vises på alle sider? Burde isåfall ha den i app, evt parent til alle filer den skal være i. Så kan du bruke et `<router-view>` for å si at routeren skal rendre der

Reply:
Det er vel planen ja, iaf ifølge figma-tegningene. Der den blir brukt nå er i øverst i selve parent filen til companylogin som routeren rendrer